docs/api/main-config-features.md

```diff
@@ -107,26 +67,4 @@ Filter args with a "target" on the type from the render function.
   ]}
 />
 
-<!-- prettier-ignore-end -->
-
-## On-demand story loading
-
-As your Storybook grows, it gets challenging to load all of your stories performantly, slowing down the loading times and yielding a large bundle. Out of the box, Storybook loads your stories on demand rather than during boot-up to improve the performance of your Storybook. If you need to load all of your stories during boot-up, you can disable this feature by setting the `storyStoreV7` feature flag to `false` in your configuration as follows:
-
-<!-- prettier-ignore-start -->
-
-<CodeSnippets
-  paths={[
-    'common/main-config-features-story-store-v7.js.mdx',
```

Comment:
Doing a Find in project for this snippet filename also yields these areas to also be removed:

- https://github.com/storybookjs/storybook/blob/next/docs/configure/index.md?plain=1#L57-L58
- https://github.com/storybookjs/storybook/blob/next/docs/configure/index.md?plain=1#L134-L154
- https://github.com/storybookjs/storybook/blob/next/docs/configure/webpack.md?plain=1#L54-L68

There's also this:

- https://github.com/storybookjs/storybook/blob/next/docs/migration-guide.md?plain=1#L123-L136

But the better move there would be updating the entire guide to be `8.0`-focused, which should probably happen in another PR.

-----

`storyStoreV7` and `buildStoriesJson` also have multiple additional instances throughout the docs that should be removed.

Reply:
cc @valentinpalkovic